mesonbuild/build.py

```diff
@@ -2418,6 +2438,11 @@ def process_kwargs(self, kwargs):
         # Visual Studio module-definitions file
         self.process_vs_module_defs_kw(kwargs)
 
+        # OS/2 uses a 8.3 name for a DLL
+        shortname = kwargs.get('shortname')
+        if shortname:
+            self.shortname = shortname
```

Comment:
Is there a reason you can't set this when you initially set `self.shortname`, like `self.shortname = kwargs.get('shortname')`?

Reply:
No problem. Maybe I followed the coding style around.